### PR TITLE
[new release] eio-ssl (0.2.0)

### PR DESCRIPTION
**Pull request by anmonteiro**
OpenSSL binding to EIO

- Project page: <a href="https://github.com/anmonteiro/eio-ssl">https://github.com/anmonteiro/eio-ssl</a>

##### CHANGES:

- Adapt to `eio` v0.10 ([anmonteiro/eio-ssl#14](https://github.com/anmonteiro/eio-ssl/pull/14),
  [anmonteiro/eio-ssl#15](https://github.com/anmonteiro/eio-ssl/pull/15))

### CODE REVIEW

File: packages/eio-ssl/eio-ssl.0.2.0/opam [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "-p"
+    name
+    "-j"
+    jobs
+    "@install"
+    "@runtest" {with-test}
+    "@doc" {with-doc}
+  ]
+]
+dev-repo: "git+https://github.com/melange-re/melange-compiler-libs.git"
```
Comment (haochenx) [inline packages/eio-ssl/eio-ssl.0.2.0/opam:29]: do you mean the following?
```suggestion
dev-repo: "git+https://github.com/anmonteiro/eio-ssl.git"
```
(it seems that the upstream might need to be updated as well
https://github.com/anmonteiro/eio-ssl/blob/5927222848f981235b658e026183a5b606a379f8/dune-project#L14-L15 )

Comment (anmonteiro) [reply packages/eio-ssl/eio-ssl.0.2.0/opam:29]: Clearly copied from somewhere else. Nice catch.

**Comment by anmonteiro** [comment]
addressed that in https://github.com/ocaml/opam-repository/pull/23913/commits/8c0b7c34921f2613058ad388ecc0b5feba8801d0

**Review by haochenx — APPROVED** [review]
looks good to me

**Comment by avsm** [comment]
Thanks! You may want to announce this on <https://discuss.ocaml.org>, where we have a Community category and an `announce` tag for this purpose.

**Pull request merged**